src/util/randomColor.js

```diff
@@ -0,0 +1,2 @@
+export const randomColor = () =>
+  '#' + Math.floor(Math.random() * 16777215).toString(16);
```

Comment:
A uitl folder usually holds small helpers that aren't directly tied to a component